lib/matplotlib/table.py

```diff
@@ -146,6 +146,42 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class SciCell(Cell):
+    """
+    A SciCell is a Cell, but it only draws the horizontal lines of the Cell
+
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
You don't actually need the `__init__` here if all you do is call the base class `__init__`